client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx

```diff
@@ -119,7 +119,7 @@ export function DomainCodePair( {
 		if ( shouldReportError && ! valid && message ) {
 			recordTracksEvent( 'calypso_domain_transfer_domain_error', {
 				domain,
-				error: message,
+				error: String( message ), // @todo some errors are React components, which break here.
```

Comment:
@escapemanuele I saw you added this and just wanted to ping you to make sure you are aware of that change.

This ensures that we are always casting to string, to avoid breaking this code if anything else was passed. `message` can have HTML elements and can be React components.

Thanks!